.circleci/config.yml

```diff
@@ -12,15 +12,14 @@ commands:
       - checkout
       - restore_cache:
           keys:
-            - v1-dependencies-{{ checksum "package.json" }}-{{ checksum "yarn.lock" }}
-            - v1-dependencies-{{ checksum "package.json" }}
-            - v1-dependencies-
+            - v2-dependencies-{{ checksum "package.json" }}-{{ checksum "yarn.lock" }}
+            - v2-dependencies-{{ checksum "package.json" }}
+            - v2-dependencies-
```

Comment:
Incrmenting the version number so that we don't pick an old cache with `.git` being cached.